calico-enterprise/networking/egress/egress-gateway-aws.mdx

```diff
@@ -850,6 +851,12 @@ spec:
 
 :::
 
+### Configure iptables backend for egress gateways
+
+The operator automatically configures egress gateways to use the same iptables backend as calico-node.
```

Comment:
Let's try something like this:

The Tigera Operator configures egress gateways to use the same iptables backend as `calico-node`. To modify the iptables backend for egress gateways, you must change the `iptablesBackend` field in the [Felix configuration](../../reference/felix/configuration.mdx).